features/admins/admin_invitations.feature

```diff
@@ -235,7 +235,87 @@ Feature: Admin Actions to Manage Invitations
       And I uncheck "Invite from queue enabled (People can add themselves to the queue and invitations are sent out automatically)"
       And I press "Update"
       And I am logged out as an admin
-    When I go to the home page
-    Then I should not see "Get Invited!"
+     When I go to the home page
```

Comment:
It looks like the spacing went wonky in these new commits, starting with the changes here. "And" steps should be indented two spaces deeper than the corresponding "When" or "Then."